README.md

```diff
@@ -1 +1,50 @@
-# Space-Travellers
\ No newline at end of file
+# Space-Travellers
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Additional description about the project and its features.
+
+This project was built to add books, remove books, update books and comment on the stored books.
+
```

Comment:
- [ ] Kindly change the description of this project since this is not a bookstore app.